diff --git a/node_modules/http-proxy/lib/http-proxy/index.js b/node_modules/http-proxy/lib/http-proxy/index.js
index 977a4b3..4d93545 100644
--- a/node_modules/http-proxy/lib/http-proxy/index.js
+++ b/node_modules/http-proxy/lib/http-proxy/index.js
@@ -121,7 +121,7 @@ ProxyServer.prototype.onError = function (err) {
   }
 };
 
-ProxyServer.prototype.listen = function(port, hostname) {
+ProxyServer.prototype.listen = function(port, hostname, callback) {
   var self    = this,
       closure = function(req, res) { self.web(req, res); };
 
@@ -133,7 +133,7 @@ ProxyServer.prototype.listen = function(port, hostname) {
     this._server.on('upgrade', function(req, socket, head) { self.ws(req, socket, head); });
   }
 
-  this._server.listen(port, hostname);
+  this._server.listen(port, hostname, callback);
 
   return this;
 };
